javascript/components/FormInput.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class ContactField extends Component {
+  render () {
+    let formElt;
+    const { inputType, inputValue, handleChange, fieldName, options } = this.props;
```

Comment:
`handleChange` should be called `onChange`--by React convention, event functions are prefixed with `on`, and the callbacks that handle them are prefixed with `handle`. In this case,

```jsx
const { onChange } = this.props;
```

and

```jsx
<ContactField
  onChange={this.handleChange}
/>
```

Reply:
Very good. Thanks! I'll implement these changes.